packages/cli/src/commands/start.ts

```diff
@@ -239,6 +239,10 @@ export class Start extends BaseCommand {
 			const taskRunnerModule = Container.get(TaskRunnerModule);
 			await taskRunnerModule.start();
 		}
+
+		if (this.globalConfig.insights.enabled) {
```

Comment:
maybe instead of this check we can have a generic list of all the modules that need to be loaded, and then we loop through that array, require-ing all of them?

Reply:
let me know what you think of this way to load modules